src/components/IncidentInfo/IncidentInfo.js

```diff
@@ -22,11 +33,15 @@ const IncidentInfo = ({ id }) => {
         {console.log(incident)}
         <p data-cy='info-name' className='info info-name'>{incident.name}</p>
         <p data-cy='info-type' className='info type'><span className='info-type'>{getIconByType(incident.incident_type)}</span>{incident.incident_type}</p>
-        <a href={mapURL} target="_blank" rel="noopener noreferrer" className='map-btn'>Click to get Map to Headquarters</a>
```

Comment:
Is this deleted from the code?

Reply:
We're good to go here. I wrapped the new button, and the map button, inside a container so I can flex them into a row.